jvm/sargon-android/src/test/java/com/radixdlt/sargon/TransactionManifestTest.kt

```diff
@@ -1,55 +1,4368 @@
 package com.radixdlt.sargon
 
+import com.radixdlt.sargon.extensions.createFungibleToken
+import com.radixdlt.sargon.extensions.createFungibleTokenWithMetadata
+import com.radixdlt.sargon.extensions.createMultipleFungibleTokens
+import com.radixdlt.sargon.extensions.createMultipleNonFungibleTokens
+import com.radixdlt.sargon.extensions.createNonFungibleToken
+import com.radixdlt.sargon.extensions.faucet
+import com.radixdlt.sargon.extensions.fromInt
+import com.radixdlt.sargon.extensions.hexToBagOfBytes
 import com.radixdlt.sargon.extensions.init
+import com.radixdlt.sargon.extensions.markingAccountAsDAppDefinitionType
+import com.radixdlt.sargon.extensions.modifyAddGuarantees
+import com.radixdlt.sargon.extensions.modifyLockFee
+import com.radixdlt.sargon.extensions.perAssetTransfers
+import com.radixdlt.sargon.extensions.perRecipientTransfers
+import com.radixdlt.sargon.extensions.setOwnerKeysHashes
+import com.radixdlt.sargon.extensions.stakesClaim
 import com.radixdlt.sargon.extensions.string
-import com.radixdlt.sargon.sample.sample
+import com.radixdlt.sargon.extensions.thirdPartyDepositUpdate
+import com.radixdlt.sargon.extensions.toDecimal192
+import com.radixdlt.sargon.samples.Sample
+import com.radixdlt.sargon.samples.sample
 import org.junit.jupiter.api.Assertions.assertEquals
 import org.junit.jupiter.api.Assertions.assertNotEquals
+import org.junit.jupiter.api.Disabled
 import org.junit.jupiter.api.Test
 
-class TransactionManifestTest {
+class TransactionManifestTest: SampleTestable<TransactionManifest> {
```

Comment:
I won't tell you to remove the assertion against instructions string, but I chose to not do it in Swift, since it is quite "expensive" to maintain, and heavily tested in Rust side. So I chose to go with `manifest.matches("token")` etc, i.e. no full string equals at all, but just "probing" that the manifest contains some expected key words. Which is much more easily maintained.